lib/shopify-cli/script/infrastructure/GraphQLBuilder.ts

```diff
@@ -0,0 +1,253 @@
+import { writeFileSync } from "fs";
```

Comment:
How much do we see this typescript tooling changing over time? For our node apps we took the approach of moving language/project-specific logic to [a node package](https://github.com/Shopify/app-cli-node-generator-helper) so that it can be version-locked and updated out of band with the CLI.

Reply:
I would love to see this stuff follow the same pattern of keeping project specific code to a package of it's own, if possible.

Reply:
Yes, we do have some discussions within the team about versioning of such tools, will make improvements in the future for sure.